install/pihole-install.sh

```diff
@@ -44,6 +44,7 @@ msg_ok "Installed Pi-hole"
 
 read -r -p "Would you like to add Unbound? <y/N> " prompt
 if [[ ${prompt,,} =~ ^(y|yes)$ ]]; then
+  read -r -p "Should Unbound be configured as a forwarding DNS server (using DNS-over-TLS (DoT)) as opposed to a recursive DNS server <y/N> " prompt
```

Comment:
```suggestion
  read -r -p "Unbound is configured as a recursive DNS server by default, would you like it to be configured as a forwarding DNS server (using DNS-over-TLS (DoT)) instead? <y/N> " prompt
```

Reply:
Yes, I tested by performing dig queries from another host, tailing the container logs, and verifying that the dns connections within the container are over port 853 with netstat (ala net-tools).